app/models/taxon.rb

```diff
@@ -23,13 +23,13 @@ def tagged_content_count(content_id)
     RummagerSearch.new(content_id).search_results_count
   end
 
+  def base_path
+    "/#{@base_path}"
```

Comment:
Why has base_path been made public here? Is it needed in a later commit?

Reply:
It is used in the view to get the base path for the 'See More' link to a tagged content list page
https://github.com/alphagov/govuk-topic-pages-prototype/blob/tagged-content-list-page/app/views/topics/show.html.erb#L41